MapboxAndroidDemo/src/main/AndroidManifest.xml

```diff
@@ -387,14 +386,15 @@
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="com.mapbox.mapboxandroiddemo.MainActivity"/>
         </activity>
-
         <activity
             android:name=".examples.camera.RestrictCameraActivity"
             android:label="@string/activity_camera_restrict_title">
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="com.mapbox.mapboxandroiddemo.MainActivity"/>
         </activity>
+        <activity android:name=".labs.BasicExtrusionActivity">
```

Comment:
This looks like more of a styles example rather then a labs. also:
```xml
android:label="@string/activity_camera_restrict_title">
            <meta-data
                android:name="android.support.PARENT_ACTIVITY"
                android:value="com.mapbox.mapboxandroiddemo.MainActivity"/>
```
seems to be missing